.changeset/chilled-seals-appear.md

```diff
@@ -0,0 +1,7 @@
+---
+"wrangler": minor
+---
+
+feature: add observability setting to wrangler.toml
+
+Adds a setting to enable automatic observability to Workers that can be viewed in the dashboard.
```

Comment:
Once the draft PR for docs is up I'll update this changeset to be more descriptive to match with the docs.